config/environments/production.rb

```diff
@@ -40,10 +40,9 @@
 
   # Assume all access to the app is happening through a SSL-terminating reverse proxy.
   # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
-  # config.assume_ssl = true
-
+  config.assume_ssl = ENV["DISABLE_SSL"].blank?
```

Comment:
```suggestion
  config.assume_ssl = ENV.include?["ASSUME_SSL"]
```

Reply:
Resolving this so we consolidate into one comment.